diff --git a/node_modules/react-native-dotenv/index.js b/node_modules/react-native-dotenv/index.js
index 148baba..80b95ec 100644
--- a/node_modules/react-native-dotenv/index.js
+++ b/node_modules/react-native-dotenv/index.js
@@ -93,10 +93,12 @@ module.exports = (api, options) => {
   env = (options.safe) ? safeObjectAssign(undefObjectAssign(undefObjectAssign(undefObjectAssign(parsed, modeParsed), localParsed), modeLocalParsed), dotenvTemporary, ['NODE_ENV', 'BABEL_ENV', options.envName])
     : undefObjectAssign(undefObjectAssign(undefObjectAssign(undefObjectAssign(parsed, modeParsed), localParsed), modeLocalParsed), dotenvTemporary)
 
-  api.addExternalDependency(path.resolve(options.path))
-  api.addExternalDependency(path.resolve(localFilePath))
-  api.addExternalDependency(path.resolve(modeFilePath))
-  api.addExternalDependency(path.resolve(modeLocalFilePath))
+  if (typeof api.addExternalDependency === 'function') {
+    api.addExternalDependency(path.resolve(options.path))
+    api.addExternalDependency(path.resolve(localFilePath))
+    api.addExternalDependency(path.resolve(modeFilePath))
+    api.addExternalDependency(path.resolve(modeLocalFilePath))
+  }
 
   return ({
     name: 'dotenv-import',
